web/models/stream.ex

```diff
@@ -21,6 +23,13 @@ defmodule Streamr.Stream do
     |> validate_required([:title])
   end
 
+  def duration_changeset(model) do
+    duration = Timex.to_unix(Timex.now()) - Timex.to_unix(model.inserted_at)
```

Comment:
A hesitation came to mind with this but after looking at the UI, I think this will be fine! It was worrying about going to the record/pressing record but we should be gucci with this.